contrib/add_license_to_files.jl

```diff
@@ -71,8 +71,9 @@ const old_license = ""
 ### END CONFIG HERE
 
 
-function check_lines!(path::AbstractString, lines::Vector, checktxt::AbstractString,
-                                                 prefix::ASCIIString, oldcheck::Bool)
+function check_lines!(
+    path::AbstractString, lines::Vector, checktxt::AbstractString,
+    prefix::AbstractString, oldcheck::Bool)
```

Comment:
I must say I don't like it either, and it doesn't look very common in the code base.